enterprise/tools/check_metrics/check_metrics.go

```diff
@@ -95,7 +103,21 @@ func main() {
 					}
 
 					if unhealthyCount >= maxUnhealthyCount {
-						return errors.New(fmt.Sprintf("%s metrics unhealthy.", metric.Name))
+						err = func() error {
+							log.Warningf("Metric %s has been consecutively unhealthy %d times. Marking as failed.", metric.Name, unhealthyCount)
+							sendMetricUnhealthySlack(metric.Name, metric.IsSecondaryMetric, config.MaxSecondaryMetricFailureCount)
```

Comment:
WDYT about gathering the status of all metrics and sending them all (including the passing ones) to slack instead of sending them off one-by-one like this?

Reply:
Done